modules/datastore/src/Storage/DatabaseTable.php

```diff
@@ -163,7 +163,7 @@ public function setSchema($schema) {
    *
    * @todo Note that this will breakZ on PostgresSQL
```

Comment:
Just a slight pre-existing grammatical error.
```suggestion
   * @todo Note that this will break on PostgresSQL
```